### PR TITLE
Recommender

### PR DESCRIPTION
**Pull request by zhekaiii**

### CODE REVIEW

File: src/features/Recommender/api/generateRecommendations.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+export const generateMockRecommendation = (placeIds: string[]) =>
+  discoverPlaces().then((places) => places[0]);
+
+export const generateRecommendation = async (placeIds: string[]) => {
+  const params = new URLSearchParams();
+  placeIds.forEach((placeId, index) => {
+    params.set(`place_id_${index + 1}`, placeId);
+  });
+  const response = await api.get<Place[]>("/v1/recommend", { params });
```
Comment (YaleChen299) [inline src/features/Recommender/api/generateRecommendations.ts:13]: not sure if i should change the backend or I missed something. the current api is `/api/v1/recommend/?place_id_1=...`

File: src/features/Recommender/api/generateRecommendations.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  return response.data[0];
```
Comment (YaleChen299) [inline src/features/Recommender/api/generateRecommendations.ts:14]: Do we want to show a few? or at least show one main one and show more when user scroll down?
If not I will try to update the backend for the option of supplying only one.

The reason that I wanted to recommend more is that, this process is not inherently perfect, so the first might not be very accurate or representative? if the user provides 5 cuisines maybe we can also show more diverse recommendations?

of course I am open for using this as MVP and we tune later!

**Review by YaleChen299 — COMMENTED** [review]
Thanks for the work! provided a few comments

File: src/features/Recommender/api/generateRecommendations.ts (hunk 1 above)
Comment (zhekaiii) [reply src/features/Recommender/api/generateRecommendations.ts:13]: Yes haha this does exactly that. `params` is the search params/query params

File: src/features/Recommender/api/generateRecommendations.ts (hunk 2 above)
Comment (zhekaiii) [reply src/features/Recommender/api/generateRecommendations.ts:14]: Yeah, I think definitely worth discussing more, but initially we discussed to show only one so as to help indecisive people decide 😂

**Review by SethCKL — APPROVED** [review]
Tested locally! works great :)

LGTM!

File: src/features/Recommender/api/generateRecommendations.ts (hunk 2 above)
Comment (SethCKL) [reply src/features/Recommender/api/generateRecommendations.ts:14]: definitely there is a sweet spot, more than 1 but less than X. 

That should be determined and tested in the future, balancing along side technical limations

**Pull request merged**